app/src/server/api/routers/datasetEntries.router.ts

```diff
@@ -71,6 +72,13 @@ export const datasetEntriesRouter = createTRPCRouter({
             "de.updatedAt as updatedAt",
             "de.outdated as outdated",
             "de.datasetId as datasetId",
+            jsonArrayFrom(
```

Comment:
This seems good enough; probably will only have O(1) relabel requests per entry anyway so not worth optimizing.